README.md

```diff
@@ -131,10 +131,17 @@ Summarizing release manager steps, at a high level, for changes into this reposi
 5. Merge the project's repository code
 6. Re-run CI in this repository, it will use the latest tag and build logic to test the build
 7. Build tarballs, and create a detached signature with the release key
-8. Commit these tarballs in the `tarballs/` directory
-9. Perform a final review and merge of the PR to this repository
-10. Tag this repository whenever releasing any packages, to verify integrity prior to building
-11. Observe nightlies the next day to ensure *all* packages are built properly
+8. Save and publish your build logs to the `build-logs` repository, e.g. https://github.com/freedomofpress/build-logs/commit/fc0eb9551678c8f58ea0017f1eb291375ea5bd9e
```

Comment:
"Publish" seems ambiguous to me - could explicitly say to checkout `main`, add the build log in the appropriate directory, and commit it.

Reply:
oh yeah "publish" here mean "merge into the main branch" since it's just a README.txt. i'll be more explicit.